test/test.ts

```diff
@@ -1,5 +1,26 @@
-describe('this package', () => {
-  it('should have tests', () => {
-    // Write your test
-  });
+import { getRelHref } from '../src/user/service';
+import { expect } from 'chai';
+
+describe('URI Test', () => {
```

Comment:
The description should probably just be the name of the function you are testing.

Then when you run the test, the output will say "getPathName should pass with a full url passed in"